astronomer/providers/google/cloud/sensors/gcs.py

```diff
@@ -78,3 +89,79 @@ def execute_complete(self, context, event=None):  # pylint: disable=unused-argum
             raise AirflowException(event["message"])
         self.log.info("File %s was found in bucket %s.", self.object, self.bucket)
         return event["message"]
+
+
+class GCSObjectsWithPrefixExistenceSensorAsync(GCSObjectsWithPrefixExistenceSensor):
+    def __init__(
```

Comment:
Docstring and comment missing explaining the sensor.

Reply:
it is inherited from sync version added doc string for only async inherited from base operator

Reply:
It still needs to be added as we have one more kwarg: `polling_interval` that's not in the inherited operator. And this will be need for our Sphinx docs